ospfd/ospf_packet.c

```diff
@@ -3409,6 +3420,13 @@ ospf_hello_send_sub (struct ospf_interface *oi, in_addr_t addr)
 
   op->dst.s_addr = addr;
 
+  if (IS_DEBUG_OSPF_TRACE)
```

Comment:
The purpose of the log is for daemon internal activity and provides different filter level of logs. If any objection will remove it.